cmd/filegen/main.go

```diff
@@ -73,6 +74,11 @@ VERSION:
 		Usage: "Metachain consensus group size",
 		Value: 15,
 	}
+	numAdditionalAccountsInGenesis = cli.IntFlag{
```

Comment:
not clear, why additional?
what will these additional accounts be used for?

Reply:
Because they won't be included in PEM files. Only in genesis.json and accounts.json